src/schema/v2/users/updateUserMutation.ts

```diff
@@ -0,0 +1,58 @@
+import { GraphQLNonNull, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import { snakeCase } from "lodash"
+
+interface Input {
+  id: string
+  email: string
+}
+
+interface GravityInput {
+  id: string
+  email: string
+}
+
+interface GravityError {
+  statusCode: number
+  body: { error?: string; text?: string; message?: string }
+}
+
+export const updateUserMutation = mutationWithClientMutationId<
+  Input,
+  any | null,
+  ResolverContext
+>({
+  name: "UpdateUserMutation",
+  description: "Update the user",
+  inputFields: {
+    id: { type: new GraphQLNonNull(GraphQLString) },
+    email: { type: new GraphQLNonNull(GraphQLString) },
+  },
+  outputFields: {},
+  mutateAndGetPayload: async (args, { updateUserLoader }) => {
+    if (!updateUserLoader) {
+      throw new Error(
+        "You need to be signed in as an admin to perform this action"
+      )
+    }
+
+    const gravityOptions = Object.keys(args)
+      .filter((key) => key !== "id")
+      .reduce(
+        (acc, key) => ({ ...acc, [snakeCase(key)]: args[key] }),
+        {} as GravityInput
+      )
+
+    try {
+      return await updateUserLoader?.(args.id, gravityOptions)
+    } catch (err) {
+      if ("body" in (err as any)) {
+        const e = err as GravityError
+        throw new Error(e.body.text ?? e.body.error ?? e.body.message)
+      }
```

Comment:
@jonallured - curious how you landed on this?

Reply:
LOL I just ripped off @dzucconi on this one:

https://github.com/artsy/metaphysics/blob/main/src/schema/v2/me/updateMyPasswordMutation.ts#L45-L66

Reply:
Ok good. find damon, copy 😄

Reply:
haha pretty much!